ui/src/components/yaml/index.tsx

```diff
@@ -419,6 +412,15 @@ const Yaml = (props: IProps) => {
                     <Markdown>{interpret}</Markdown>
                   )}
                 </div>
+                {interpretStatus === 'streaming' && interpret && (
```

Comment:
The inline ternary operator for determining the class name makes the code harder to read and maintain. Consider extracting this logic into a separate function or variable for better readability.